### PR TITLE
u-boot: set u-boot sources to xentroops repository

### PR DESCRIPTION
**Pull request by oleksiimoisieiev**
Set u-boot sources to the repository with support an enhanced support of the RPI5 devices.

### CODE REVIEW

File: recipes-bsp/u-boot/u-boot-tools_2024.04.bb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-require recipes-bsp/u-boot/u-boot.inc
+3require recipes-bsp/u-boot/u-boot.inc
```
Comment (lorc) [inline recipes-bsp/u-boot/u-boot-tools_2024.04.bb]: `3require`? :)

**Comment by dsemenets** [comment]
I think SRC_URI and SRCREV should be moved to separate file

File: recipes-bsp/u-boot/u-boot-tools_2024.04.bb (hunk 1 above)
Comment (oleksiimoisieiev) [reply recipes-bsp/u-boot/u-boot-tools_2024.04.bb]: yep - it's remove already )))

**Comment by oleksiimoisieiev** [comment]
> I think SRC_URI and SRCREV should be moved to separate file

Good idea. Reworked

**Review by lorc — APPROVED** [review]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by dsemenets** [comment]
`Reviewed-by: Dmytro Semenets <dmytro_semenets@epam.com>`

**Pull request merged**